packages/core/src/components/popover-canvas/popover-canvas.tsx

```diff
@@ -38,6 +38,9 @@ export class TdsPopoverCanvas {
   /** Array of modifier objects to pass to popper.js. See https://popper.js.org/docs/v2/modifiers/ */
   @Prop() modifiers: Object[] = [];
 
+  /** Decides if the popover should disable its internal logic. Leaving selector or referenceEl empty will disable logic too. */
+  @Prop() disableLogic: boolean = false;
+
```

Comment:
Just a suggestion, I would instead look into creating logic for controlled vs uncontrolled states. 

Separate the props to defaultOpen and open. 

If none of the props are used we do all the opening/closing in the component (uncontrolled.)

If a user adds a false/true (not undefined) value for `open` prop we never do any internal logic and just opens/closes the Popover when the open prop changes value. (controlled)

If a users set defaultOpen to true the Popover is open by default and we do all the open/closing in the component (uncontrolled)

This would mirror Reacts uncontrolled vs controlled pattern. See: https://www.freecodecamp.org/news/what-are-controlled-and-uncontrolled-components-in-react/

If I don't want to control this state I would do:

```jsx
<tds-popover selector={'#selectorId'}>
   // My popover content
</tds-popover>
``` 

If I don't want to control this state, but I want it open by default I would do:

```jsx
<tds-popover selector={'#selectorId'} default-open>
   // My popover content
</tds-popover>
``` 

If I as a user wants to control the open/close state I would do: 

```jsx
<tds-popover open={myTrueOrFalseVaraibleThatWillUpdateInRuntime}>
   // My popover content
</tds-popover>
``` 

For the controlled one I would recommend exposing a tdsOpenChange event that fires when a user presses outside the Popover (so when it would close), and the user can listen to this and decide if they want to close the Popover.

Reply:
Sorry for the long comment. But i think this would be a good approach for all interactive components. To try and think of how a users could either control it fully, not control it at all or maybe just set one initial state.